tests/erlang_tests/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 # SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 #
 
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required (VERSION 2.8.12)
```

Comment:
We don't need 3.13 here but maybe we should say 3.13 everywhere?

Reply:
For the record, on FreeBSD 12:
```
$ uname -a
FreeBSD dev 12.2-RELEASE-p12 FreeBSD 12.2-RELEASE-p12 ec84e0c52a1(HEAD) TRUENAS  amd64
$ cmake --version
cmake version 3.22.2

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

Reply:
We need 3.13 for target_link_options, the newest command used now.
Fortunately, debian stretch-backports has 3.13 (but not a newer version).